### DISCUSSION TITLE
add algerians tv

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#269] [Show and tell]
this site has a live stream of algerians tv , when i extract a stream link 'mpd ' dont work !
https://iptv.at.dz/
can some one help me

**Comment by BellezaEmporium** [comment]
GET on https://itvepg2.at.dz/iptvepg/frame1039/action/sdk_gettvplayurl.jsp?channelcode=ch00000000000000001021&columncode=0101&definition=1&mediaservice=2&authid=720218&mediaservice=2&urlredirect=1&needjitp=1&jitpmediatype=DASH&jitpdrmtype=Widevine&playtype=0&timestamp=1641043806074

with channel code, timestamp changing over time.
media can be DASH or HLS, DRM if HLS doesn't need to be filled up.

**Reply by BellezaEmporium** [reply]
EDIT : Found out they were using basically a Chinese host (ZTE OTT) to get their own channels, which is a weird choice and counter productive.

Channel names doesn't seem to correspond with data I have from other Chinese hosts. I don't know if China Mobile has got those channels, let me check.
